javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var exceptStrings = [];
+  for (var i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This loop is trying to filter out strings, not select specifically numbers 🤔. What if the array included objects or arrays or null etc?

Reply:
fixed!